spec.bs

```diff
@@ -2104,6 +2081,19 @@ An auction config is a [=struct=] with the following items:
 
 </dl>
 
+<div algorithm="validate and convert per buyer signals">
+
+To <dfn>validate and convert per buyer signals</dfn> given an {{AuctionAdConfig}} |auctionConfig| and a JavaScript value |value|:
+1. If |value| is not a [=record=] mapping from {{USVString}} to {{any}}, throw a {{TypeError}}.

```

Comment:
|value| is now [guaranteed](https://webidl.spec.whatwg.org/#dfn-perform-steps-once-promise-is-settled) to be a `record<USVString, any>`, so you don't need to check it. If it were still a plain JS value, you'd use `Let |record| be the result of [=converted to an IDL value|converting=] |value| to record<USVString, any>` (https://webidl.spec.whatwg.org/#dfn-convert-ecmascript-to-idl-value), and that operation would throw a TypeError if the conversion failed.
```suggestion
To <dfn>validate and convert per buyer signals</dfn> given an {{AuctionAdConfig}} |auctionConfig|
and a `record<USVString, any>` |value|:
```

Reply:
Inlined again (w/o the check); some of the other stuff may end up out-of-line though; is the placement OK?